index.js

```diff
@@ -115,14 +115,71 @@ const parseArgs = (args, extraTypesForMessage) => {
   return { message, data }
 }
 
+const formatData = (data) => {
+  if (simpleTypes.includes(typeof data)) 
+    return `${stringifySimple(data, DEPTH)}`
+
+  if (typeof data == 'object')
```

Comment:
```suggestion
  if (typeof data === 'object')
```

Reply:
Fixed in https://github.com/DeversiFi/logger/pull/5/commits/eb79cf40d4e5cf56ca634d81cba3ca0140674d14